src/test/regress/expected/isolation_replicate_reference_tables_to_coordinator.out

```diff
@@ -102,13 +118,14 @@ step s2-view-worker:
     WHERE query NOT ILIKE '%pg_prepared_xacts%' AND
           query NOT ILIKE '%COMMIT%' AND
           query NOT ILIKE '%dump_local_%' AND
-          query NOT ILIKE '%citus_internal_local_blocked_processes%'
+          query NOT ILIKE '%citus_internal_local_blocked_processes%' AND
+          query NOT ILIKE '%add_node%'
     ORDER BY query, query_hostport DESC;
 
-query                                                                         |query_hostname|query_hostport|distributed_query_host_name|distributed_query_host_port|state              |wait_event_type|wait_event|usename |datname
+query                                                                        |query_hostname|query_hostport|distributed_query_host_name|distributed_query_host_port|state              |wait_event_type|wait_event|usename |datname
 ---------------------------------------------------------------------
-UPDATE public.ref_table_1500777 ref_table SET a = (a OPERATOR(pg_catalog.+) 1)|localhost     |         57638|coordinator_host           |                      57636|idle in transaction|Client         |ClientRead|postgres|regression
-UPDATE public.ref_table_1500777 ref_table SET a = (a OPERATOR(pg_catalog.+) 1)|localhost     |         57637|coordinator_host           |                      57636|idle in transaction|Client         |ClientRead|postgres|regression
+UPDATE public.ref_table_1500777 ref_table SET a = (a OPERATOR(pg_catalog.+) 1)|localhost     |         57638|                           |                          0|idle in transaction|Client         |ClientRead|postgres|regression
```

Comment:
Well,. this is very hard to fix. isolation test somehow keeps using the same session and the impact of add_node doesn't show up. I'm ok to skip for now